source/_integrations/point.markdown

```diff
@@ -23,6 +23,7 @@ ha_integration_type: integration
 The Point hub enables integration with the [Minut Point](https://minut.com/). To connect with Point, you will have to [sign up for a developer account and have a Pro subscription](https://minut.com/community/developers/) and get a `client_id` and `client_secret` with the `callback url` configured as your Home Assistant URL + `/api/minut`, e.g.,  `http://localhost:8123/api/minut`. The `client_id` and `client_secret` should be used as below.
 
 Once Home Assistant is started, a configurator will pop up asking you to Authenticate your Point account via a link. When you follow the link and click on **Accept** you will be redirected to the `callback url` and the Point integration will be automatically configured and you can go back to the original dialog and press **Submit**.
+The link provided needs to be tweaked: replace 'localhost:8000/point?' with 'https://hassurl:8123/api/minut?'; leave the rest of the link untouched. It seems to work best in MS Edge.
```

Comment:
I'm not following what you are trying to add/change here.

In the first paragraph, the callback is already explained in the first paragraph, what does it add on this place? And what is up with MS Edge? Doesn't it work on other browsers? If so, have you reported that as an issue?

Reply:
Hello @frenck, thanks for the quick reply.

The first paragraph is not clear on what steps to follow, so with these extra explaination it will work. (It took me a while, by googling the right steps) That way these steps explain better what to do.
I have not reported it as an issue yet, will do it now also. Noticed I used other browsers which didn't work, so I found a solution to do it with MS Edge.

Reply:
> That way these steps explain better what to do.

I'm sorry, I don't agree. For me it feels like a duplicate at the end, which is dangling a bit without a clear path.

> I have not reported it as an issue yet, will do it now also. Noticed I used other browsers which didn't work, so I found a solution to do it with MS Edge.

Ok, but please report those cases. We should not document bugs.